### PR TITLE
separate connection for egress

### PR DESCRIPTION
**Pull request by adiweiss**
issue: https://github.com/wework/grabbit/issues/46

### CODE REVIEW

File: gbus/bus.go [hunk 1]
```diff
@@ -29,6 +29,7 @@ type DefaultBus struct {
 	PrefetchCount  uint
 	AmqpConnStr    string
 	amqpConn       *amqp.Connection
```
Comment (rhinof) [inline gbus/bus.go]: let's rename this variable to ingressConn

File: gbus/bus.go [hunk 2]
```diff
@@ -29,6 +29,7 @@ type DefaultBus struct {
...
+	outAmqpConn    *amqp.Connection
```
Comment (rhinof) [inline gbus/bus.go]: rename to egressConn

File: gbus/bus.go [hunk 3]
```diff
@@ -29,6 +29,7 @@ type DefaultBus struct {
...
 	workers        []*worker
 	AMQPChannel    *amqp.Channel
```
Comment (rhinof) [inline gbus/bus.go]: rename to ingressChannel

File: gbus/bus.go [hunk 4]
```diff
@@ -29,6 +29,7 @@ type DefaultBus struct {
...
 	outAMQPChannel *amqp.Channel
```
Comment (rhinof) [inline gbus/bus.go]: rename to egressChannel

File: gbus/bus.go (hunk 1 above)
Comment (adiweiss) [reply gbus/bus.go]: all fixed

**Pull request merged**